### PR TITLE
Update PerfCounters.cc

### PR DESCRIPTION
**Pull request by 20urc3**
Fix bug #3770

### CODE REVIEW

File: src/PerfCounters.cc [hunk 1]
```diff
@@ -100,6 +100,7 @@ enum CpuMicroarch {
   AMDZen2,
   AMDZen3,
   AMDZen4,
+  AMDRyzen9,
```
Comment (rocallahan) [inline src/PerfCounters.cc:103]: Don't need to add this; this CPU is Zen 4.

**Review by rocallahan — CHANGES_REQUESTED** [review]

File: src/PerfCounters.cc (hunk 1 above)
Comment (20urc3) [reply src/PerfCounters.cc:103]: Try it. Not supported.

Comment (rocallahan) [reply src/PerfCounters.cc:103]: Why did you put 0x6f there? That's the `llsc_cntr_event` and it makes no sense to set that for an x86 CPU.

Using Zen4 should work. If it doesn't, something else is quite broken.

**Review by rocallahan — CHANGES_REQUESTED** [review]

File: src/PerfCounters.cc (hunk 1 above)
Comment (20urc3) [reply src/PerfCounters.cc:103]: I've just followed the pattern I saw before. After compiling this version it works on my end, so maybe the issue is not fix by my did but was already fixed between the latest package release and the git version?

Comment (rocallahan) [reply src/PerfCounters.cc:103]: Yeah, this change doesn't actually do anything because nothing detects this new architecture type AMDRyzen9..

**Pull request closed without merging**